package.json

```diff
@@ -22,7 +22,8 @@
     "packages/*"
   ],
   "scripts": {
-    "bootstrap": "lerna bootstrap && lerna link",
+    "bootstrap": "cross-env LUMOS_NODE_RUNTIME=electron LUMOS_NODE_RUNTIME_VERSION=7.1.4 lerna bootstrap && lerna link",
+    "prebootstrap": "npm i cross-env@5.2.1 --no-save",
```

Comment:
`npx` can do this work without another script.

Reply:
I think it is a nice approach to go with `npx` in this case.

I had a try with `npx -c 'cross-env...` in the script `bootstrap`, but it doesn't seem to work. The child script `rebuild:nativemodules`  of `postinstall` throws an error at the end.

Please feel free to push the code or suggest if you know how to achieve this with `npx`.

Reply:
Don't use `-c` with `npx`, I got an error with that kind of usage.
And it reminds me that the `env` should be limited in the neuron-wallet, better in the `rebuild:neodeps` script as the following:
```sh
"rebuild:neondeps": "npx cross-env LUMOS_NODE_RUNTIME=electron LUMOS_NODE_RUNTIME_VERSION=7.1.4 electron-build-env neon build -p ../../node_modules/@ckb-lumos/indexer --release"
```

Reply:
Thanks for the example.

As we are using lumos indexer. 0.4.8, which comes with a prebuilt binary for electron 7, we don't need to build it anymore. The reason we need to have these env variables `LUMOS_NODE_RUNTIME=electron LUMOS_NODE_RUNTIME_VERSION=7.1.4` is to tell the lumos npm installation script to download the electron version of binary instead of the default node version. They need to be passed on to the `lerna bootstrap`, which in turn passes on to the lumos indexer npm installation script. 

To make these env variables compatible with different OS, it uses `cross-env`. In a clean setup, `cross-env` executable won't be available when running `yarn bootstrap`, thus we need to figure out a way to make sure `cross-env` is installed. I had another try with the `npx` without `-c` so that we don't need `prebootstrap` script, and it seems working now.